MovieApp/Infrastructure/Data/DataSource/MovieService.swift

```diff
@@ -0,0 +1,142 @@
+import Foundation
+
+class MovieService {
```

Comment:
Great job on creating a Service class. To push this even further, I would suggest separating each of the models (MovieResponse, MovieDetailsResponse, CrewMember, Criteria) to a separate file. And then, having a `MovieServiceProtocol` used as a function blueprint would be a cherry on the top